modules/twitch-notify/events/botReady.js

```diff
@@ -0,0 +1,52 @@
+const {confDir} = require('../../../main');
+const config = require(`${confDir}/twitch-notify/config.json`);
+
+const { ApiClient } = require('twitch');
+const { ClientCredentialsAuthProvider } = require('twitch-auth');
+
+const ClientID = config['Twitch-ClientID'];
+const ClientSecret = config['ClientSecret'];
+const authProvider = new ClientCredentialsAuthProvider(ClientID, ClientSecret);
+const apiClient = new ApiClient({ authProvider });
+
+let live = false;
+
+function twitch_notify(client) {
+  function replacer(msg, username, game) {
+    msg = msg.split('%streamer%').join(username)
+        .split('%game%').join(game)
+        .split('%url%').join('https://twitch.tv/' + username.toLowerCase());
+    return msg
+  };
+
+  function sendMSG(username, game) {
```

Comment:
Please name this function `sendMsg` as "msg" does not contain more than one word